app/src/main/java/fr/neamar/kiss/result/SettingsResult.java

```diff
@@ -53,6 +53,7 @@ public void doLaunch(Context context, View v) {
         if (!settingPojo.packageName.isEmpty()) {
             intent.setClassName(settingPojo.packageName, settingPojo.settingName);
         }
+		intent.setSourceBounds(v.getClipBounds());
```

Comment:
I think this method is only available on Android 18+, so this would be a problem on 14-17 that we're still supporting